### ISSUE TITLE
NativeMethodsMixin is undefined

### ISSUE THREAD

**Issue by TimIJpenga** [shoutem/animation#7]
When using React 15.4.0 or higher, `Parallax.js` cannot find `NativeMethodsMixin`.
Since React 15.4.0 they have moved `NativeMethodsMixin `from the React package to the React Native package. [see here](https://github.com/facebook/react/pull/7168)

I managed to fix it by simply changing this in `Parallex.js`
```
import React, { Component, } from 'react';
import NativeMethodsMixin from 'react/lib/NativeMethodsMixin';
import { Animated, View, Dimensions } from 'react-native';
```

to this
```
import React, { Component, } from 'react';
import { Animated, View, Dimensions, NativeMethodsMixin } from 'react-native';
```

I can create a pull request to fix this, however I dont know how this is going to affect older versions of react native or react.

**Comment by SoHotSoup** [comment]
Hey @TimIJpenga!
You can create a pull request, there are several ways to address it. You can either change `peerDependencies`, if you replace `^` with `~` on react's version or if newer React Native versions are dependent on `react 15.4.0`, then change import as you already suggested and upgrade react-native version (inside our package.json) to one that has `NativeMethodsMixin` included.

Thank you for giving us support on this issue!

**Comment by SoHotSoup** [comment]
If you need any help, we can do it together!

**Comment by TimIJpenga** [comment]
Hi @SoHotSoup 
When I get home in about 4 hours I will look at it and see what it does with different versions.
I will let you know when I need help, thank you.

**Comment by SoHotSoup** [comment]
Cool! Thank you!

**Comment by SoHotSoup** [comment]
Fixed by #8
